### ISSUE TITLE
protractor?

### ISSUE THREAD

**Issue by el-davo** [gonzofish/angular-librarian#56]
Hi great app, We have adopted it as our main scaffolding app for generating libraries. Today we updated it to support protractor as we use protractor alot. Is this something you would be willing to accept a pull request on? If not then thats fine to :+1:

**Comment by gonzofish** [comment]
Definitely! I've never done protractor in a serious manner, but I'd be willing to bring it in.

And it's great to hear that you guys have found this library useful!
